templates/templates/experiment.template

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
ditto on moving to a subdir for preview templates

Reply:
changed